api/uninstall/uninstall.go

```diff
@@ -27,6 +27,7 @@ type Request struct {
 	DryRun       bool   `json:"dry_run"`
 	KeepHistory  bool   `json:"keep_history"`
 	DisableHooks bool   `json:"disable_hooks"`
+	Timeout      int64  `json:"timeout"`
```

Comment:
Have kept the timeout parameter numeric instead of string since it will be easier to use numeric value programmatically

Reply:
@punit-kulal This can just be `int` right?